ansys/content.jsonc

```diff
@@ -10,6 +10,7 @@
     "long_desc": [
         "Ansys suite of computational simulation products are available to DesignSafe users, including the structural analysis and computational fluid dynamics modules. Most simulations are performed using the Ansys Workbench system.",
         "TACC's Ansys license is available only for non-commercial, academic use. To access Ansys please <a href=\"https://www.designsafe-ci.org/help/new-ticket/\">submit a ticket</a> and provide your institutional affiliation and a brief statement confirming that you will use Ansys only for non-commercial, academic purposes."
+        "Note: Ansys is only accessible by logging to TACC's supercomputers via <a href=\"https://tap.tacc.utexas.edu/\">TACC Analysis Portal (TAP)</a> or Secure Shell (SSH). More details on how to use Ansys on TACC can be found by clicking the User Guide button below."
```

Comment:
**Great!** Tweaks:

```suggestion
        "Ansys is only accessible by logging into TACC's supercomputers via <a href=\"https://tap.tacc.utexas.edu/\" target=\"_blank\">TACC Analysis Portal (TAP)</a> or Secure Shell (SSH). From there, you can charge your DesignSafe allocation."
```

1. I think we need not tell user where the button is, because (a) the very next section is "Learn How to Use Ansys" and (b) I'll rename the section to "Learn How to Use Ansys on TACC".
2. @cs-jansen proposes appending, "From there, you can charge your DesignSafe allocation."